symbolic-minidump/Cargo.toml

```diff
@@ -29,9 +29,13 @@ include = [
 [package.metadata.docs.rs]
 all-features = true
 
+[features]
+default = ["processor"]
```

Comment:
I would prefer a feature name that makes the dependency on breakpad clearer. Maybe `breakpad-processor` or something like that.

Reply:
I originally wanted to just name the feature `breakpad`. But that is essentially opaque. We don’t have a second processor implementation in symbolic, so turning off breakpad, you turn off the whole processor. And that is also in line with symbolicator, since it won’t be using the symbolic crate as processor anymore.